eo-runtime/src/main/java/org/eolang/Dataized.java

```diff
@@ -104,8 +104,9 @@ public Object take() {
             Dataized.LOGGER.log(
                 Level.FINE,
                 String.format(
-                    "%s\uD835\uDD3B( %s ) ➜ %s",
+                    "%s\uD835\uDD3B( <%s>%s ) ➜ %s",
```

Comment:
@mximp maybe let's get rid of phi-term here at all and only print the location? the logging will be much more compact and readable

Reply:
@yegor256 The problem is that location isn't always available in runtime due to the way attributes are retrieved: `PhLocated` is excluded from the call chain. We will see `<?:?>` in half (if not more) of the cases.
The output is not that huge with this implementation anyway and is readable (since we don't go into attributes recursively).
We might want to raise a separate issue for re-designing attributes retrieval logic to have location in every Phi.